lib/actions/local/configure.coffee

```diff
@@ -95,100 +67,65 @@ inquirerOptions = (data) -> [
 	}
 ]
 
+joinPath = (path) -> '/' + path.join('/')
+
+load = (target, type) ->
```

Comment:
Can we rename this function to something a bit more descriptive (e.g. `loadReconfix`, or something else)